app/decorators/valkyrie/resource_decorator.rb

```diff
@@ -1,5 +1,25 @@
 # frozen_string_literal: true
 class Valkyrie::ResourceDecorator < ApplicationDecorator
+  self.suppressed_attributes = [
```

Comment:
I'm not sure I understand what's going on here, since at first blush these look like attributes we'd want to display.  Is this related to moving the imported attributes to a nested resource, so these get suppressed on the parent resource?